packages/components/src/tabs/test/index.tsx

```diff
@@ -1625,52 +1187,35 @@ describe( 'Tabs', () => {
 		} );
 		describe( 'Disabled tab', () => {
 			it( 'should render the specified `selectedTabId` (not the first enabled tab) when the tab associated to `initialTabId` is disabled', async () => {
```

Comment:
I'm not sure about the point of this test — `initialTabid` should not be used in controlled mode anyway (and should be always ignored in favor of `selectedTabId` )